scripts/main/build.js

```diff
@@ -252,36 +250,52 @@ build.photo = function (data, disabled = false) {
 };
 
 build.overlay_image = function (data) {
-	let exifHash = data.make + data.model + data.shutter + data.aperture + data.focal + data.iso;
-
 	// Get the stored setting for the overlay_image
 	let type = lychee.image_overlay_type;
-	let html = ``;
+	let html = lychee.html`
+				<div id="image_overlay">
+				<h1>$${data.title}</h1>
+				`;
+	if (data.takedate && data.takedate !== "")
+		html += `<p>${data.takedate}</p>
+				`;
+	else
+		html += `<p>${data.sysdate}</p>
+				`;
 
 	if (type && type === "desc" && data.description !== "") {
-		html = lychee.html`
-					<div id="image_overlay">
-						<h1>$${data.title}</h1>
-						<p>$${data.description}</p>
-					</div>
+		html += `<p>$${data.description}</p>
 				`;
 	} else if (type && type === "takedate" && data.takedate !== "") {
-		html = lychee.html`
-			<div id="image_overlay">
-				<h1>$${data.title}</h1>
-				<p>${data.takedate}</p>
-			</div>
+		html += `<p>$${data.takedate}</p>
```

Comment:
Won't this result in the `takedate` being displayed twice, given your unconditional addition of `takedate`/`sysdate` under the title?